templates/ast-iam-roles.template.yaml

```diff
@@ -165,6 +165,8 @@ Resources:
       ManagedPolicyArns:
         - !Sub 'arn:${AWS::Partition}:iam::aws:policy/CloudWatchAgentServerPolicy'
         - !Sub 'arn:${AWS::Partition}:iam::aws:policy/AmazonSSMManagedInstanceCore'
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/service-role/AWSLambdaVPCAccessExecutionRole"
```

Comment:
This role is assumed by EC2 and thus should not have a Lambda execution policy

Reply:
Ah yes good catch